packages/react-native/ReactAndroid/src/main/java/com/facebook/debug/debugoverlay/model/DebugOverlayTag.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package com.facebook.debug.debugoverlay.model
+
+import javax.annotation.concurrent.Immutable
+
+/** Tag for a debug overlay log message. Name must be unique.  */
+@Immutable
+public class DebugOverlayTag public constructor(
```

Comment:
Thought about making this a data class, wasn't sure if it was needed though. Will leave it up to the reviewer to decide